READMEs/c2d-flow.md

```diff
@@ -0,0 +1,398 @@
+-   Python 3.8.5+
+
+### Run barge services
+
+Ocean `barge` runs ganache (local blockchain), Provider (data service), and Aquarius (metadata cache).
+
+In a new console:
+
+```console
+#grab repo
+git clone https://github.com/oceanprotocol/barge
+cd barge
+
+#clean up old containers (to be sure)
+docker system prune -a --volumes
+
+# make sure to use the dev version of the operator service URL, to match the barge provider
+export OPERATOR_SERVICE_URL=https://c2d-dev.operator.oceanprotocol.com/
+
+#run barge: start ganache, Provider, Aquarius; deploy contracts; update ~/.ocean
+./start_ocean.sh  --with-provider2
+```
+
+### Install the library
+
+In a new console that we'll call the work console (as we'll use it later):
+
+```console
+#Initialize virtual environment and activate it.
+python -m venv venv
+source venv/bin/activate
+
+#Install the ocean.py library. Install wheel first to avoid errors.
+pip install wheel
+pip install ocean-lib
+```
+
+This example uses c2d to create a regression model. In order to visualise it or manipulate it, you also need some dependencies:
+
+```console
+pip install numpy
+pip install matplotlib
+```
+
+### Create config file
+
+In the work console:
+```console
+#Create config.ini file and fill it with configuration info
+echo """
+[eth-network]
+network = http://127.0.0.1:8545
+address.file = ~/.ocean/ocean-contracts/artifacts/address.json
+
+[resources]
+metadata_cache_uri = http://localhost:5000
+provider.url = http://localhost:8030
+provider.address = 0x00bd138abd70e2f00903268f3db08f2d25677c9e
+
+downloads.path = consume-downloads
+""" > config.ini
+```
+
+### Set envvars
+
+In the work console:
+```console
+#set private keys of two accounts
+export TEST_PRIVATE_KEY1=0x5d75837394b078ce97bc289fa8d75e21000573520bfa7784a9d28ccaae602bf8
+export TEST_PRIVATE_KEY2=0xef4b441145c1d0f3b4bc6d61d29f5c6e502359481152f869247c7a4244d45209
+
+#needed to mint fake OCEAN for testing with ganache
+export FACTORY_DEPLOYER_PRIVATE_KEY=0xc594c6e5def4bab63ac29eed19a134c130388f74f019bc74b8f4389df2837a58
+
+```
+
+### Config in Python
+
+In the work console:
+```console
+python
+```
+
+For the following steps, we use the Python console. Keep it open between steps.
+
+In the Python console:
+```python
+#create ocean instance
+from ocean_lib.config import Config
+from ocean_lib.ocean.ocean import Ocean
+config = Config('config.ini')
+ocean = Ocean(config)
+
+print(f"config.network_url = '{config.network_url}'")
+print(f"config.metadata_cache_uri = '{config.metadata_cache_uri}'")
+print(f"config.provider_url = '{config.provider_url}'")
+
+# Create Alice's wallet
+import os
+from ocean_lib.web3_internal.wallet import Wallet
+alice_wallet = Wallet(ocean.web3, os.getenv('TEST_PRIVATE_KEY1'), config.block_confirmations)
+print(f"alice_wallet.address = '{alice_wallet.address}'")
+
+# Mint OCEAN for ganache only
+from ocean_lib.ocean.mint_fake_ocean import mint_fake_OCEAN
+mint_fake_OCEAN(config)
```

Comment:
You don't need this. This is only in the market README as liquidity for the AMMs. This README doesn't have that.